src/react_native/core.cljs

```diff
@@ -50,3 +55,23 @@
   (js->clj (.get (.-Dimensions ^js react-native) "window") :keywordize-keys true))
 
 (def status-bar (.-StatusBar ^js react-native))
+
+(def navigation-const (atom nil))
```

Comment:
please also remove  59 -77 lines, because its not needed for this PR ,i'll migrate it later properly, thanks